### PR TITLE
Update busy_start/busy_stop ui events documentation

### PR DESCRIPTION
**Pull request by glacambre**

### CODE REVIEW

File: runtime/doc/ui.txt [hunk 1]
```diff
@@ -219,8 +219,9 @@ the editor.
 
 ["busy_start"]
 ["busy_stop"]
-	Nvim started or stopped being busy, and possibly not responsive to
-	user input. This could be indicated to the user by hiding the cursor.
+	This event is misnamed and does not actually have anything to do with
```
Comment (bfredl) [inline runtime/doc/ui.txt:222]: I would reverse the order of these sentences (start with telling the reader what to actually do :)

**Comment by glacambre** [comment]
@bfredl I performed the change you suggested, is there anything else needed before this PR can be merged? :)

File: runtime/doc/ui.txt [hunk 2]
```diff
@@ -219,8 +219,8 @@ the editor.
 
 ["busy_start"]
 ["busy_stop"]
-	Nvim started or stopped being busy, and possibly not responsive to
-	user input. This could be indicated to the user by hiding the cursor.
+	Indicates to the UI that it must stop rendering the cursor. This event
+	is misnamed and does not actually have anything to do with business.
```
Comment (janlazo) [inline runtime/doc/ui.txt:223]: https://www.thefreedictionary.com/busyness ?

Comment (glacambre) [reply runtime/doc/ui.txt]: Woops, thanks for catching that!

**Comment by janlazo** [comment]
Thanks @glacambre .

**Pull request merged**